apps/server/src/modules/learnroom/mapper/room-board-response.mapper.ts

```diff
@@ -47,6 +47,7 @@ export class RoomBoardResponseMapper {
 			createdAt: boardTask.createdAt,
 			updatedAt: boardTask.updatedAt,
 			status: boardTaskStatus,
+			taskCard: boardTask.taskCard,
```

Comment:
put the following line separately after const mappedTask...

```suggestion
			if (boardTask.taskCard) {
			   mappedTask.taskCardId = boardTask.taskCard;
```